.github/workflows/release.yaml

```diff
@@ -29,7 +30,9 @@ jobs:
           username: ${{ vars.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
       - name: Push Helm chart
-        run: make helm-push HELM_CHART_VERSION=${HELM_CHART_VERSION}
+        run: |
+          make helm-push HELM_CHART_VERSION=${HELM_CHART_VERSION}
+          make helm-push HELM_CHART_VERSION=${HELM_CHART_VERSION_WITHOUT_V}
```

Comment:
I think to be consistent, we also should make the same change to https://github.com/envoyproxy/ai-gateway/blob/895d8958f05b10d5c0ac81ede94724b36a2f477a/.github/workflows/tests.yaml#L212 for the latest v0.0.0

Reply:
Fair, updated.